src/service_name.json

```diff
@@ -0,0 +1,514 @@
+  },
+  {
+    "Command": "az hanainstance",
+    "Description": "Manage Azure SAP HANA instance (preview)",
+    "AzureServiceName": "SAP HANA on Azure",
+    "ConversationalServiceName": "SAP HANA on",
+    "ExtensionName": "sap-hana",
+    "URL": "https://docs.microsoft.com/azure/virtual-machines/workloads/sap/hana-overview-architecture"
+  },
+  {
+    "Command": "az healthbot",
+    "Description": "Manage Azure Bot service with healthbot",
+    "AzureServiceName": "Azure Bot",
+    "ConversationalServiceName": "Bot",
+    "ExtensionName": "healthbot",
+    "URL": "https://docs.microsoft.com/azure/bot-service/"
+  },
+  {
+    "Command": "az healthcareapis",
+    "Description": "Microsoft Azure Command-Line Tools HealthcareApisManagementClient Extension",
+    "AzureServiceName": "Azure API for FHIR",
+    "ConversationalServiceName": "API for FHIR",
+    "ExtensionName": "healthcareapis",
+    "URL": "https://docs.microsoft.com/azure/healthcare-apis/overview"
+  },
+  {
+    "Command": "az hpc-cache",
+    "Description": "Microsoft Azure Command-Line Tools StorageCache Extension",
+    "AzureServiceName": "Azure HPC Cache",
+    "ConversationalServiceName": "HPC Cache",
+    "ExtensionName": "hpc-cache",
+    "URL": "https://docs.microsoft.com/azure/hpc-cache/hpc-cache-overview"
+  },
+  {
+    "Command": "az import-export",
+    "Description": "Microsoft Azure Command-Line Tools StorageImportExport Extension",
+    "AzureServiceName": "Azure Storage",
+    "ConversationalServiceName": "Storage",
+    "ExtensionName": "import-export",
+    "URL": "https://docs.microsoft.com/azure/storage/common/storage-import-export-service?toc=/azure/storage/blobs/toc.json"
+  },
+  {
+    "Command": "az interactive",
+    "Description": "Start interactive mode. Installs the Interactive extension if not installed already.",
+    "AzureServiceName": "Azure CLI",
+    "ConversationalServiceName": "Azure CLI",
+    "ExtensionName": "interactive",
+    "URL": "https://docs.microsoft.com/cli/azure/"
+  },
+  {
+    "Command": "az internet-analyzer",
+    "Description": "Microsoft Azure Command-Line Tools Internet Analyzer Extension",
+    "AzureServiceName": "Azure Network",
+    "ConversationalServiceName": "Network",
+    "ExtensionName": "internet-analyzer",
+    "URL": "https://docs.microsoft.com/azure/internet-analyzer/"
+  },
+  {
+    "Command": "az k8s-configuration",
+    "Description": "Manage Kubernetes configurations",
+    "AzureServiceName": "Azure Kubernetes",
+    "ConversationalServiceName": "Kubernetes Service",
+    "ExtensionName": "k8s-configuration",
+    "URL": "https://docs.microsoft.com/azure/aks/"
+  },
+  {
+    "Command": "az k8sconfiguration",
+    "Description": "Microsoft Azure Command-Line Tools K8sconfiguration Extension",
+    "AzureServiceName": "Azure Kubernetes Service",
+    "ConversationalServiceName": "Kubernetes Service",
+    "ExtensionName": "k8sconfiguration",
+    "URL": "https://docs.microsoft.com/azure/aks/"
+  },
```

Comment:
I think `k8s-configuration` and `k8sconfiguration` are the same.

Reply:
Unfortunately, they are two extension. Ugly naming.
```
k8s-configuration               1.0.0      Microsoft Azure Command-Line Tools K8s-configuration Extension                                                     False      False           False
k8sconfiguration                0.2.4      Microsoft Azure Command-Line Tools K8sconfiguration Extension                                                      True       False           False
```